packages/react-art/src/ReactART.js

```diff
@@ -75,6 +84,11 @@ class Surface extends React.Component {
       false,
       false,
       '',
+      defaultOnUncaughtError,
+      defaultOnCaughtError,
+      defaultOnRecoverableError,
```

Comment:
These were missing in ART so any errors would've logged like `onUncaughtError is undefined` instead of the error. Probably not heavily used.